tests/distinct/tnil.nim

```diff
@@ -0,0 +1,41 @@
+discard """
+  file: "tnil.nim"
+  output: "0x1\n\nnil\n\nnil\n\n"
```

Comment:
I like the obscured triple quotes feature together with black box newline normalization. Makes me feel like in the 90s using ASCII Art :)